src/coreclr/minipal/Unix/doublemapping.cpp

```diff
@@ -23,6 +23,18 @@
 
 #if defined(TARGET_OSX) && defined(TARGET_AMD64)
 #include <mach/mach.h>
+#include <sys/sysctl.h>
+
+bool IsProcessTranslated()
+{
+   int ret = 0;
+   size_t size = sizeof(ret);
+   if (sysctlbyname("sysctl.proc_translated", &ret, &size, NULL, 0) == -1)
+   {
+      return false;
+   }
+   return ret != 0;
```

Comment:
nit: in other places, we have used `== 1`:

https://github.com/dotnet/runtime/blob/051b4828c7d3a0cad3289830ef9fd2120f45bb2b/src/native/libs/System.Native/pal_runtimeinformation.c#L112
https://github.com/dotnet/runtime/blob/051b4828c7d3a0cad3289830ef9fd2120f45bb2b/src/native/corehost/hostmisc/pal.unix.cpp#L1090

Reply:
Thank you @am11 for the info, I didn't know we already check for this somewhere.